File: ExamPreparation1/JavaSkeleton/src/main/java/imdb/entity/Film.java
package imdb.entity;

import javax.persistence.*;

@Entity
@Table(name = "films")
public class Film {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;

    private Integer year;

    private String name;

    private String director;

    private String genre;

    public Film(Integer year, String name, String director, String genre) {
        this.year = year;
        this.name = name;
        this.director = director;
        this.genre = genre;
    }

    public Film() {
    }

    public Integer getId() {
        return id;
    }

    public Integer getYear() {
        return year;
    }

    public String getName() {
        return name;
    }

    public String getDirector() {
        return director;
    }

    public String getGenre() {
        return genre;

    }

    public void setYear(Integer year) {
        this.year = year;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setDirector(String director) {
        this.director = director;
    }

    public void setGenre(String genre) {
        this.genre = genre;
    }
}

File: ExamPreparation1/PHPSkeleton/src/AppBundle/Entity/Film.php
<?php

namespace AppBundle\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Film
 *
 * @ORM\Table(name="films")
 * @ORM\Entity(repositoryClass="AppBundle\Repository\FilmRepository")
 */
class Film
{
    /**
     * @var int
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;

    /**
     * @var int
     * @ORM\Column(name="year", type="integer")
     */
    private $year;

    /**
     * @var string
     * @ORM\Column(name="director", type="string")
     */
    private $director;

    /**
     * @var string
     * @ORM\Column(name="name", type="string")
     */
    private $name;

    /**
     * @var string
     * @ORM\Column(name="genre", type="string")
     */
    private $genre;

    /**
     * @param int $year
     * @return Film;
     */
    public function setYear($year)
    {
        $this->year = $year;
        return $this;
    }

    /**
     * @param string $director
     * @return Film
     */
    public function setDirector($director)
    {
        $this->director = $director;
        return $this;
    }

    /**
     * @param string $name
     * @return Film
     */
    public function setName($name)
    {
        $this->name = $name;
        return $this;
    }

    /**
     * @param string $genre
     * @return Film
     */
    public function setGenre($genre)
    {
        $this->genre = $genre;
        return $this;
    }

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @return int
     */
    public function getYear()
    {
        return $this->year;
    }

    /**
     * @return string
     */
    public function getDirector()
    {
        return $this->director;
    }

    /**
     * @return string
     */
    public function getName()
    {
        return $this->name;
    }

    /**
     * @return string
     */
    public function getGenre()
    {
        return $this->genre;
    }



}

